### ISSUE TITLE
Someone can update the "Credits" page?

### ISSUE THREAD

**Issue by SarcrofS** [clintbellanger/flare-game#532]
Someone can update the "Credits" page?

I love some assets from this game but most of the assets aren't posted in Credits page, so I don't  know how find the "original" and what license are. Exemple: who make the traps? what are the license of this?

I need know what is the license for flying loot of new weapons too (I think Dorkster made, but don't find the license), Magic Channeling Spell Effect and Traps Effect (spikes).

Some licenses was changed too (thanks God) to CC-0 and CC-BY from CC-BY-SA.
If I remember well, this was the changes of listed assets:
Musics: Magical Theme and Creepy Forest come to CC-BY 3.0.
Sound FX: Ice  Spells, Spells 4 (fire) and 25 spooky sound effects come to CC-BY 3.0.

Thanks!

**Comment by dorkster** [comment]
The Magic Channeling Spell Effect and the Traps Effect are both by @clintbellanger if I'm correct. They're either licensed CC-BY or CC-BY-SA, so it's best to ask him.

I made the flying loot animation for the new weapons, but I'd be fine with keeping the attribution to Clint there. Not only did he make the weapons, but I used his existing loot animations as a guideline for creating mine, so it's like 99% his work.

I updated the licenses for the other assets you mentioned, with the exception of "Spells 4 (fire)". That one is multi-licensed, so I just left it as CC-BY-SA 3.0.
